src/mgr/Mgr.cc

```diff
@@ -344,6 +344,10 @@ void Mgr::shutdown()
   // give up the lock for us.
   Mutex::Locker l(lock);
 
+  monc->sub_unwant("log-info");
+  monc->sub_unwant("mgrdigest");
+  monc->sub_unwant("fsmap");
```

Comment:
I worked around this with https://github.com/ceph/ceph/pull/14578/commits/54a29f2174ef00864357f2570dfea6166bc23f1d#diff-bf7f5cbe59b62fb0a72436135106b3f5R178 but unwant is probably better!